.github/workflows/update-readthedocs.yml

```diff
@@ -59,9 +59,15 @@ jobs:
             exit 1
           fi
 
+          # Get the version from the GitHub ref
+          VERSION=${GITHUB_REF#refs/*/}
```

Comment:
Isn't the ref always going to be `refs/heads/main` since the flow is triggered on a push to main?  What's the different with the current "latest" then? Am I missing something?

Don't we need to update the flow with:

```
on:
  push:
    tags:
      - '*'
```

Reply:
Updated